### ISSUE TITLE
devbio-napari-env for multi-users

### ISSUE THREAD

**Issue by grinic** [haesleinhuepf/devbio-napari#57]
Hi @haesleinhuepf , thanks a lot for your work on this great package!

I work in an imaging facility, and would like to install devbio-napari-env for all users in a read-only environment.

I have installed miniforge for all users (it gets installed on C:\conda\miniforge3), and I managed to successfully create other environments in read-only mode (e.g. cellpose).

However, when I run
`conda create --name devbio-napari-env napari=0.4.19 python=3.9 devbio-napari pyqt -c conda-forge -c pytorch`
I am only able to activate `devbio-napari-env` and open `naparia` when I am in an administrator command window. 

In a normal command window, activating the environment gives:

```
H:\>conda activate devbio-napari-env
Access is denied.
The system cannot find the file specified.
Could Not Find C:\conda\miniforge3\envs\devbio-napari-env\Library\etc\OpenCL\vendros\temp.txt

(devbio-napari-env) H:\>
```

It seems the environment is activated anyhow.
However, running `naparia` gets stuck and never opens the napari interface.

Any idea what might go wrong here? In cellpose environemnt, I had to modify the users right to allow writing access in the __ pycache __ folder, but in this case this also doesn't work.

Thanks a lot for the help!

**Comment by haesleinhuepf** [comment]
Hi @grinic ,

great idea! Thanks for reaching out! I've been working in a similar core facility for some years and we struggled often with users modifying installations - and breaking stuff for other people. That's why I recommend not installing Python, Napari or Fiji in a way that multiple users can access it. However, there is a way out: When a Windows user logs into a system for the first time, you can copy apps to their folder, where they have write-access. Let's see if git-bob knows how to do this:

git-bob comment on how to achive this: I would like to configure Windows in a way that whenever a user logs into the system for the first time, some apps are copied to their profile, e.g. Fiji, Napari, Python, and they can modify the installation without the risk to screw things up for others using the same computer, because the apps are stored in their user space. How can I do this?

**Comment by haesleinhuepf** [comment]
Hey @grinic ,

do you think this might work?

Best,
Robert

**Comment by grinic** [comment]
Thanks for the tip! I will have a look and see if we can set it up properly.
In general though, we are working on virtual machines that do not have a lot of disk space (data access is directly to the servers), so copying the folder for every user doesn't seem like a scalable solution?

**Comment by haesleinhuepf** [comment]
Yes, it might require some disk space. Yet again: If one user installs a plugin to Fiji or Napari, which breaks it, all users will suffer, if they work with the same installation where everyone has write-access.
